lineage/states/stateCommon.py

```diff
@@ -18,7 +18,7 @@
 def nLL_sep(x, uncens_obs, uncens_gammas, cens_obs, cens_gammas):
     a, scale = x
     uncens = jnp.dot(uncens_gammas, gamma.logpdf(uncens_obs, a=a, scale=scale))
-    cens = jnp.dot(cens_gammas, gammaincc(a, cens_obs / scale))
+    cens = jnp.dot(cens_gammas, jnp.log(gammaincc(a, cens_obs / scale)))
```

Comment:
Are you absolutely sure you need this change? If not, remove it.

Reply:
The upper one is in the log form, and the lower one was not. And we are summing them together. I think we should keep it this way.

Reply:
This is not an acceptable justification for changing the equation. You need to look back at the paper Shakthi found for this and confirm the log is needed. Alternatively, if you see it definitely improves the fitting results, I might be convinced it's correct based on that alone.

Reply:
Either way I'd suggest removing this from the current PR, because the rest of the changes generally look good.

Reply:
removed.